contracts/query-queue/tests/integration.rs

```diff
@@ -182,7 +182,7 @@ fn create_integrated_query_contract() -> (Instance<MockApi, MockStorage, MockQue
                     .with_storage(|storage| VmResult::Ok(storage.get(key).0.unwrap()))
                     .unwrap();
                 if data.is_none() {
-                    return SystemResult::Err(SystemError::Unknown {});
+                    return SystemResult::Ok(ContractResult::Ok(Binary::from(b"null")));
```

Comment:
Wow! This return value is a bit complicated.

Reply:
Before this change, it errors if there is no value in storage bonded with the specified key. (And, it confused me for fixing 😅 )
This is not good for testing, so fix it to return the binary formed string "null".